practices/soyliz30/README.md

```diff
@@ -0,0 +1,14 @@
+Lizeth Palacios Patiño
```

Comment:
Recomiendo seguir el formato de [markdow](https://docs.github.com/es/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax)